File: BtgChallenge/BtgChallenge/Networking/Response/ListResponse/ListResponse.swift
//
//  ListResponse.swift
//  BtgChallenge
//
//  Created by Felipe Alexander Silva Melo on 15/05/20.
//  Copyright © 2020 Felipe Alexander Silva Melo. All rights reserved.
//

import Foundation

struct ListResponse: Codable {
    var success: Bool?
    var terms: String?
    var privacy: String?
    var currencies: CurrenciesResponse?
}

File: app/src/main/java/com/br/mobilechallenge/model/ListResponse.kt
package com.br.mobilechallenge.model


import com.google.gson.annotations.SerializedName

data class ListResponse(
    @SerializedName("currencies")
    val currencies: Map<String, String>,
    @SerializedName("privacy")
    val privacy: String,
    @SerializedName("success")
    val success: Boolean,
    @SerializedName("terms")
    val terms: String
)